### PR TITLE
Use namespace default to match the GIF

### PR DESCRIPTION
**Pull request by cscetbon**

### CODE REVIEW

File: 04-deny-traffic-from-other-namespaces.md [hunk 1]
```diff
@@ -62,24 +62,23 @@ Query this web service from the `default` namespace:
 
 ```sh
 $ kubectl run test-$RANDOM --namespace=default --rm -i -t --image=alpine -- sh
```
Comment (ahmetb) [inline 04-deny-traffic-from-other-namespaces.md]: you should've changed these pods to secondary, then?

**Comment by ahmetb** [comment]
Thanks for the patch. I think we might have a few other image vs content inconsistencies. I personally don't think it's a big deal. 

Looks like by flipping namespace you still ended up with issues.

File: 04-deny-traffic-from-other-namespaces.md (hunk 1 above)
Comment (cscetbon) [reply 04-deny-traffic-from-other-namespaces.md]: Correct

**Comment by cscetbon** [comment]
I don’t think that the fact I was wrong in my patch means it’s useless. I think it’s not a big deal for people who are used to network policies. But for others it’s different.

**Comment by cscetbon** [comment]
> I don’t think that the fact I was wrong in my patch means it’s useless.

If I did not understand you correctly, nvm.

**Comment by cscetbon** [comment]
@ahmetb I use now `foo` as the namespace  source for blocked queries.

File: 04-deny-traffic-from-other-namespaces.md [hunk 2]
```diff
@@ -50,36 +50,35 @@ networkpolicy "deny-from-other-namespaces" created"
...
 
 ## Try it out
 
-Query this web service from the `default` namespace:
+Query this web service from the `foo` namespace:
 
 ```sh
-$ kubectl run test-$RANDOM --namespace=default --rm -i -t --image=alpine -- sh
-/ # wget -qO- --timeout=2 http://web.secondary
+$ kubectl run test-$RANDOM --namespace=foo --rm -i -t --image=alpine -- sh
```
Comment (ahmetb) [inline 04-deny-traffic-from-other-namespaces.md]: I believe you need to create `foo` namespace first.
Also need to clean it up at the end.

File: 04-deny-traffic-from-other-namespaces.md [hunk 3]
```diff
@@ -17,12 +17,12 @@ pod deployed to.
 
 ### Example
 
-Create a new namespace called `secondary` and start a web service:
+Create a new namespace called `default` and start a web service:
```
Comment (boredabdel) [inline 04-deny-traffic-from-other-namespaces.md:20]: the default namespace usually exist on a cluster so no need to create it.

File: 04-deny-traffic-from-other-namespaces.md (hunk 2 above)
Comment (boredabdel) [reply 04-deny-traffic-from-other-namespaces.md:64]: @cscetbon please add the following line

kubectl create namespace default

before line 64

File: 04-deny-traffic-from-other-namespaces.md [hunk 4]
```diff
@@ -50,36 +50,35 @@ networkpolicy "deny-from-other-namespaces" created"
...
 
 ### Cleanup
 
-    kubectl delete deployment web -n secondary
-    kubectl delete service web -n secondary
-    kubectl delete networkpolicy deny-from-other-namespaces -n secondary
-    kubectl delete namespace secondary
+    kubectl delete deployment web -n default
+    kubectl delete service web -n default
+    kubectl delete networkpolicy deny-from-other-namespaces -n default
```
Comment (boredabdel) [inline 04-deny-traffic-from-other-namespaces.md:84]: Please add a kubectl delete namespace foo

File: 04-deny-traffic-from-other-namespaces.md (hunk 2 above)
Comment (cscetbon) [reply 04-deny-traffic-from-other-namespaces.md:64]: @boredabdel I think you mean `foo` as `default` always exists

Comment (boredabdel) [reply 04-deny-traffic-from-other-namespaces.md:64]: Yes sorry foo

**Review by boredabdel — APPROVED** [review]

**Pull request merged**